services/provider/server/server.go

```diff
@@ -621,13 +619,6 @@ func decodeAndValidateTicket(ticket string, pubKey *rsa.PublicKey) (*services.On
 	}
 
 	switch ticketData.SubjectRole {
-	case services.ClientRole:
-		if ticketData.StorageQuotaInGiB != nil {
-			quota := *ticketData.StorageQuotaInGiB
-			if quota > math.MaxInt {
-				return nil, fmt.Errorf("invalid value sent in onboarding ticket, storage quota should be greater than 0 and less than %v: %v", math.MaxInt, quota)
-			}
-		}
```

Comment:
Why?

Reply:
my mistake in removing the `case`, I should've removed only the quota check.

Reply:
fixed.